samples/nrf9160/mqtt_simple/overlay-qemu-x86-tls.conf

```diff
@@ -0,0 +1,21 @@
+# To build the mqtt simple sample with TLS support
```

Comment:
I think this information should rather be placed in the README.rst, let's add the copyright info instead.